ethereumkit/src/main/java/org/p2p/ethereumkit/external/EthereumModule.kt

```diff
@@ -32,5 +33,6 @@ object EthereumModule {
                 gson = get(named(RPC_JSON_QUALIFIER))
             )
         }
+        single { PriceCacheRepository() }
```

Comment:
I can't see any interface here, and naming is wrong
PricesInMemoryRepository() bind PricesRepository